src/display/api.js

```diff
@@ -885,14 +922,14 @@ class PDFDocumentProxy {
  * PDF page operator list.
  *
  * @typedef {Object} PDFOperatorList
- * @property {Array} fnArray - Array containing the operator functions.
- * @property {Array} argsArray - Array containing the arguments of the
- *                               functions.
+ * @property {Array<Function>} fnArray - Array containing the operator
```

Comment:
This looks strange, since there's nothing which is `typeof "function"` in the array.

Reply:
You're right. I think I just guessed based on the name. I ran an example and set the type to be `Array<number>` instead.